pkg/tools/gen/testdata/jsonschema/additional/expect.k

```diff
@@ -0,0 +1,26 @@
+"""
+This file was generated by the KCL auto-gen tool. DO NOT EDIT.
+Editing this file might prove futile when you re-run the KCL auto-gen generate command.
+"""
+
+schema Ethernet:
+    """
+    Ethernet
+
+    Attributes
+    ----------
+    name : str, optional
+    socket : Socket, optional, default is {"HTTP": 80, "HTTPS": 443}
+    """
+
+    name?: str
+    socket?: Socket = {"HTTP": 80, "HTTPS": 443}
+    [...str]: any
```

Comment:
if the index signature `[...str]: any` here will still be retained?

Reply:
fixed